packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -555,6 +556,13 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
                     if (clientIdOrDocumentId.length === 0 ||
                         clientIdOrDocumentId === this.documentId ||
                         (this.hasDetails && clientIdOrDocumentId === this.clientId)) {
+                        const nackContent = message[0]?.content;
+                        if (nackContent !== undefined) {
+                            this.logger.sendTelemetryEvent({
+                                eventName: "ServerNack",
+                                nackDetails: JSON.stringify(nackContent),
```

Comment:
If you do go the "details" route, please just call it `details`. The idea is to include this column in most queries, to catch as many events as possible.  I really want to see us have WAY fewer columns in Kusto, starting "grass roots"/ad hoc for now but maybe will incorporate some type-based helpers down the road.